core/src/autogluon/core/models/abstract/abstract_model.py

```diff
@@ -985,63 +978,48 @@ def _hyperparameter_tune(self, X, y, X_val, y_val, scheduler_options, **kwargs):
             model_cls=model_cls,
             init_params=init_params,
             time_start=time_start,
-            time_limit=scheduler_params['time_out'],
+            time_limit=hpo_executor.time_limit,
             fit_kwargs=fit_kwargs,
             train_path=train_path,
             val_path=val_path,
+            hpo_executor=hpo_executor,
+        )
+        model_estimate_memory_usage = None
+        if self.estimate_memory_usage is not None:
+            model_estimate_memory_usage = self.estimate_memory_usage(X=X, **kwargs)
+        hpo_executor.execute(
+            model_trial=model_trial,
+            train_fn_kwargs=train_fn_kwargs,
+            directory=directory,
+            minimum_cpu_per_trial=self.get_minimum_resources().get('num_cpus', 1),
+            minimum_gpu_per_trial=self.get_minimum_resources().get('num_gpus', 0.1),
```

Comment:
This parameter name is very confusing: reading it creating impression that HPO is not possible without GPU. If this is not the case, then API is lying to the user what it is really doing.